bears/shell/ShellCheckBear.py

```diff
@@ -20,8 +20,14 @@ class ShellCheckBear:
     CAN_DETECT = {'Syntax', 'Security', 'Undefined Element', 'Unused Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file, shell: str='sh'):
+    def create_arguments(filename, file, config_file, shell: str='sh',
+                         shellcheck_ignore: list=None):
         """
         :param shell: Target shell being used.
+        :param shellcheck_ignore: List of linting rules that should be ignored.
```

Comment:
It is safe to assume that in this case the user knows about shellcheck. So lets rename `shellcheck_ignore` to what it is called in shellcheck: `exclude`

https://github.com/koalaman/shellcheck/blob/22c86256ace897e2fcd780da46bc173fb1bbd5f5/shellcheck.hs#L77